### ISSUE TITLE
Enso multiline AI Integration

### ISSUE THREAD

**Issue by JanBeelte** [enso-org/enso#7382]
1. Move the Language-Server AI Endpoint functionality into the AI Visualization of the Enso Standard Library.
	1. Assemble the query (existing already)
	2. Call OpenAI API from Enso [see on discord](https://discord.com/channels/401396655599124480/1124228885467365457)
	3. Add proper error handling for REST related issues (timeouts, missing OPENAI_KEY, ...)
	4. Deprecate and Remove the Language-Server AI Endpoint.
2. Expose all available variables in scope (and their column layout) to the AI.
	1. How do we query all variables that are in scope?
	2. For each of the variables we can run the already existing column name extraction logic.
3. Support multiline and Chat results
	1. The AI Visualization needs to be refactored to output a tuple of: multiline enso code and the chat commentary. The main issue here is how the multi line enso code will be processed and displayed by the IDE. The chat commentary could be ignored in a first step.
	2. The chat commentary should be displayed either in a Chat-like interface or as a comment next to the generated code.

Task Number 1 (and potentially also 2) are mainly Enso Standard Library Tasks and should be completely written in Enso.

Task Number 3 is an IDE task that could be started in parallel by using a mock implementation that contains a fixed result and commentary (I can provide both).

**Comment by farmaazon** [comment]
Closing in favor of https://github.com/enso-org/enso/issues/9656 and different AI design.
